vortex-api/build.gradle.kts

```diff
@@ -28,6 +28,9 @@ dependencies {
     implementation("systems.uom:systems-common:2.0.2")
     implementation("edu.ucar:cdm-core:5.5.3")
     implementation("org.apache.commons:commons-compress:1.20")
+    implementation("org.hdfgroup:hdfobject:2.14.0a")
+    implementation("org.hdfgroup:jarh5obj:2.10.1")
+    implementation("org.hdfgroup:jarhdf5:3.3.2")
```

Comment:
Are all 3 of these neccesary or just `hdfobject`? I recall you saying you used just the base lib.